src/deflate/mod.rs

```diff
@@ -29,3 +27,5 @@ pub enum Deflaters {
         iterations: NonZeroU8,
     },
 }
+
+impl Copy for Deflaters {}
```

Comment:
This should be cleaner if we add it to the derives list

Reply:
Thanks! Not sure why I thought I needed an impl...